docs/release-notes/release-notes-0.18.3.md

```diff
@@ -33,6 +33,11 @@
   created](https://github.com/lightningnetwork/lnd/pull/8921) when multiple RPC
   calls are made concurrently.
 
+* [Fixed a bug](https://github.com/lightningnetwork/lnd/pull/8857) that caused
+  LND not persisting on main cfg the changes done calling the commands 
+  `lncli debuglevel` to change the debuglevel and `lncli setmccfg` to change 
+  the estimator.
```

Comment:
nit: "Correctly propagate mission control and debug level config values to the main LND config struct so that the `GetDebugInfo` response is accurate"